src/python/python.i

```diff
@@ -5,3 +5,7 @@
 %include Vector2.i
 %include Vector3.i
 %include Vector4.i
+%include Line2.i
+%include Line3.i
+%include SignalStats.i
+%include Temperature.i
```

Comment:
nit: can these be alphabetized or do they need to be ordered by dependencies?

Reply:
> do they need to be ordered by dependencies?

That's correct